tests/snuba/rules/conditions/test_event_frequency.py

```diff
@@ -89,7 +114,12 @@ class EventFrequencyQueryTest(EventFrequencyQueryTestBase):
     rule_cls = EventFrequencyCondition
 
     def test_batch_query(self):
-        condition_inst = self.rule_cls(self.event.group.project)
+        data = {"interval": "1m", "value": 6}
+        condition_inst = self.get_rule(
```

Comment:
nit: should we move these to the class level so we don't have to redefine them each test?